pkg/api/meta/meta.go

```diff
@@ -373,7 +414,8 @@ type genericAccessor struct {
 	labels            *map[string]string
 	annotations       *map[string]string
 	// TODO: This has to be a pointer to list, otherwise we can implement SetOwnerReferences
-	ownerReferences []unversioned.ObjectReference
+	// ownerReferences []*unversioned.ObjectReference
+	ownerReferences listAccessor
```

Comment:
Actually I have rewritten the it to use the pointer of the slice in the next commit. I'll squash the commit after I've addressed your other comment.